src/packages/cli/src/__tests__/__helpers__/integrationTest.ts

```diff
@@ -337,6 +339,7 @@ async function setupScenario(kind: string, input: Input, scenario: Scenario) {
         generator client {
           provider = "prisma-client-js"
           output   = "${ctx.fs.path()}"
+          previewFeatures = ["microsoftSqlServer"]
```

Comment:
Still have to figure out a way to provide this programmatically

Reply:
Yeah I'm just checking [database name](https://github.com/prisma/prisma/compare/feat/integration-tests-mssql#diff-fcd4e781e37b047303886e2ce2bcfb331688c92f2e732d1511d8387409667a72R333) right now